client/web/src/user/settings/auth/ExternalAccountsSignIn.tsx

```diff
@@ -95,6 +116,33 @@ const getNormalizedAccount = (
                     }
                 }
                 break
+            case 'SAML':
+                {
+                    const samlExternalData = accountExternalData as SamlExternalData
+                    const data = samlExternalData.Assertions[0].AttributeStatement.Attributes
+                    let nickname = ''
+
+                    for (let index = 0, length = data.length; index < length; index += 1) {
+                        const item = data[index]
+                        if (data[index].Values && item.Name) {
+                            const key = item.Name.slice(Math.max(0, item.Name.lastIndexOf('/') + 1))
```

Comment:
I think either a comment that explains how the `item.Name` is transformed or a test would go a long way here 😄

Reply:
I think we can write this as:
```suggestion
                            const key = item.Name.split('/').pop()
```

Which should make the code more readable and self-explanatory (I hope 😄 )